x-pack/plugins/secops/public/components/page/network/network_dns_table/columns.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import numeral from '@elastic/numeral';
+import { isNil } from 'lodash/fp';
+import React from 'react';
+
+import { NetworkDnsFields, NetworkDnsItem } from '../../../../graphql/types';
+import { escapeQueryValue } from '../../../../lib/keury';
+import { networkModel } from '../../../../store';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { defaultToEmptyTag, getEmptyTagValue } from '../../../empty_value';
+import { Columns } from '../../../load_more_table';
+import { Provider } from '../../../timeline/data_providers/provider';
+
+import * as i18n from './translations';
+
+export const getNetworkDnsColumns = (
+  startDate: number,
+  type: networkModel.NetworkType
+): Array<Columns<NetworkDnsItem>> => [
+  {
+    field: `node.${NetworkDnsFields.dnsName}`,
+    name: i18n.REGISTERED_DOMAIN,
+    truncateText: false,
+    hideForMobile: false,
+    sortable: true,
+    render: dnsName => {
+      const id = escapeDataProviderId(`networkDns-table--name-${name}`);
+      if (!isNil(dnsName)) {
+        return (
+          <DraggableWrapper
+            key={id}
+            dataProvider={{
+              and: [],
+              enabled: true,
+              id,
+              name: dnsName.toString(),
```

Comment:
Is this `toString()` needed? It's already a string and if you check for null above you won't have worry about the bangs here, right? Can this just be `dnsName` without `.toString()`?

Reply:
it was complaining